packages/core/usePointerSwipe/index.ts

```diff
@@ -99,7 +99,7 @@ export function usePointerSwipe(
     useEventListener(target, 'pointerdown', (e: PointerEvent) => {
       isPointerDown.value = true
       // Disable scroll on for TouchEvents
-      targetRef.value?.setAttribute('style', 'touch-action: none')
+      targetRef.value?.style.setProperty("touch-action", "none")
```

Comment:
shouldn't be?:
`targetRef.value?.style?.setProperty("touch-action", "none")`

Reply:
also use single quote
`targetRef.value?.style?.setProperty('touch-action', 'none')`

Reply:
you forget also change on `pointerup` event... https://github.com/vueuse/vueuse/blob/main/packages/core/usePointerSwipe/index.ts#L130

Reply:
K thanks, was wondering as well if on line 51
https://github.com/vueuse/vueuse/blob/main/packages/core/usePointerSwipe/index.ts#L51
the Ref should be typed as HTMLElement instead of Element because `style` doesn't always exist on Element?

Reply:
I think you should wait for author review...